app/main/views/g_cloud.py

```diff
@@ -35,8 +35,11 @@
 from app import search_api_client, data_api_client, content_loader
 
 
+END_SEARCH_LIMIT = 100  # TODO: This should be done in the API.
 PROJECT_SAVED_MESSAGE = Markup("""Search saved.""")
-PROJECT_ENDED_MESSAGE = Markup("""Your search has been ended. You can now download your shortlist.""")
+PROJECT_ENDED_MESSAGE = Markup("""Search ended. You can now download your search results.""")
```

Comment:
@karlchillmaid - is this fine for content?

Reply:
Yes, that's the content that was signed off for the flash banners.